python/lsst/ts/ofc/ctrlIntf/OFCCalculationFactory.py

```diff
@@ -51,7 +52,7 @@ def getCalculator(instName, state0Dof=None):
         Returns
         -------
         OFCCalculationOfLsst, OFCCalculationOfComCam, OFCCalculationOfSh, or
-        OFCCalculationOfCmos
+        OFCCalculationOfCmos, OFCCalculationOfLsstFam
```

Comment:
misplaced or - shall be before OFCCalculationOfLsstFam, not before OFCCalculationOfCmos. Consider replacing the list with OFCCalculation child (OFCCalculationOfComCam).

I am not sure, but my understanding is that you shall drop OFCCalculationOfLsst class, as it is now (renamed?) OFCCalculationOfLsstFam - at least drop of LSST from InstName suggest this.

Reply:
Fixed in the commit: c90538b. Thanks!